---
output: 
  pdf_document: 
    latex_engine: xelatex
header-includes:
- \usepackage[fontsize=10pt]{scrextend}
geometry: margin=1in
bibliography: "`r sharedbib::bib_path()`"
csl: nature.csl
---

```{r setup, include=FALSE}
library(readr)
library(dplyr)
library(knitr)
opts_chunk$set(echo = FALSE, message = FALSE, include = FALSE)
options(scipen=999) # Stops scientific nottaion form bengin used and breaking latex
```

```{r}
estimate_error <- 1.1
sites <- 2
species <- 9
trees <- 8
seeds <- 4 + 1 # 1 is combined sample
seed_neg_dna_controls <- 8 
seed_neg_pcr_controls <- 8 
seed_mock_controls <- 8
soil_neg_dna_controls <- 4
soil_neg_pcr_controls <- 4
soil_mock_controls <- 4
plant_neg_dna_controls <- 4
plant_neg_pcr_controls <- 4
plant_mock_controls <- 4
```


```{r}
(seed_samples <- species * trees * seeds * sites)
(soil_samples <- species * trees * sites)
(plant_samples <- species * trees * sites)
(all_controls <- (seed_neg_dna_controls +  
    seed_neg_pcr_controls +  
    seed_mock_controls + 
    soil_neg_dna_controls + 
    soil_neg_pcr_controls + 
    soil_mock_controls + 
    plant_neg_dna_controls + 
    plant_neg_pcr_controls + 
    plant_mock_controls) * sites)
(samples_sequenced <- seed_samples + soil_samples + plant_samples + all_controls)
```

# The coevolution of plant seeds and their microbiome


\newpage

## Project Summary

#### Overview

The composition of the microbiome has been shown to be an important factor for the nutrition and defense of plants [@rodriguez2008stress; @arnold2003fungal; @ernst2003endophytic; @bose1947hereditary; @latch1985endophytic; @arachevaleta1989effect; @barker1984effect1; @barker1984effect2; @clay1987effects] and animals [@rosshart2017wild; @hosokawa2006strict].
Given the immense importance of some of these interactions for basic functions [@ohkuma2003termite], plants and animals are perhaps better thought of as reproducing ecosystems or "super-organisms" [@mendes2011deciphering; @boursnell1950symbiotic], rather than simply hosts for an assortment of symbionts, parasites, and pathogens.
Plant seeds represent a highly specialized reproductive unit that has evolved to maximize the chance of survival by supplying physical protection and nutrition to a developing embryo.
If plants are integrated evolutionarily with their symbionts, then seeds might have evolved to incorporate microbial aids to defense and nutrition.
It is well known that various bacteria, fungi, and oomycetes inhabit seeds and these are often the same members that occur in mature individuals [@nelson2018seed; @bergna2018tomato].
There is also evidence suggesting that seed microbes can be vertically transmitted to the next generation by colonizing the developing seedling [@truyens2015bacterial; @ernst2003endophytic; @boursnell1950symbiotic; @bergna2018tomato].
Furthermore, some of these vertically transmitted microbes have beneficial properties [@johnston2014impact; @bergna2018tomato].
Finally, it has been shown that plants selectivity recruit microbes from the environment and that host genotype affects this ability [@wu1995molecular; @bergna2018tomato].
Given that mature plants have evolved to encourage beneficial microbes, that these microbes occur in seeds, and can be transmitted via seeds to the next generation, it seems likely that some plants have co-evolved with a symbiotic microbiome transmitted and conserved via seed.
If true, this would have important implications for the understanding of seed plants biology and for the selection and treatment of seeds in agriculture.

The primary objective of this proposal is to test for a correlation between the phylogenetic similarity of seed plants and their seed microbiomes, which would be consistent with co-evolution.
A secondary objective is to characterize the identity, distribution, and diversity of microbes inhabiting seeds of plants in two Long Term Ecological Research (LTER) sites. 
This will be accomplished by collecting seeds from natural populations of plants and comparing their phylogenetic relatedness to the phylogenetic similarity of their microbial communities while accounting for environmental differences.
Seeds will be collected from 10 conifer species in the H.J. Andrews LTER site and 10 poaceae species in the Konza Prairie LTER site.
DNA extracted from seeds will be used to characterize the phylogenetic relationship of the plants using maximum likelihood and the similarity of their microbiomes using metabarcoding, resulting in a pairwise dissimilarity matrix for the plants and another for their microbiomes.
Associated data such as soil nutrients, micro-climate, and the soil microbiome will be recorded for each plant sampled and used to make a third pairwise dissimilarity matrix.
Finally, a partial mantel test will be used to test for a correlation between the host phylogenetic dissimilarity matrix and the microbiome dissimilarity matrix, while accounting for environmental factors using the environmental distance matrix.
Additional analyses will also be conducted, including taxonomic classification of microbes, calculation of alpha and beta diversity indexes, analysis of differential taxon abundance among plant species, and indicator species analysis.

#### Intellectual merit

This study will help to determine how integrated plants are with their microbiome evolutionarily and has the potential to inform basic seed plant biology.
It will also be one of the few studies to use metabarcoding to reveal the microbial diversity of individual seeds in natural populations, avoiding the reliance on culturing needed in past studies.
Finally, it will provide valuable information on the microbial diversity in two LTER sites that can inform future studies.

#### Broader impacts

Agriculture will soon have to accommodate population increases, climate change, soil degradation, aquifer depletion, and lower availability of fossil fuels for fertilizer and pesticides.
Most major food crops are planted by seed and a better understanding of how seed-transmitted microbes function in natural systems can inform what seed treatment strategies are likely to increase food production.
In particular, if plants are found to co-evolve with their microbiome, the current practice of sterilizing seed might actually be damaging in some cases, similar to how an excessively sterile environment for young humans is now being correlated with disease in later life, such as asthma [@lambrecht2017immunology]. 

\newpage


## Project Description

### Background

In order for an adaptive seed microbiome maintinaed through vertical trasmission to be possible, the following phenomia must exist:

* Microbes assocaited with plants must provide some benifit to plant fitness.
* Plant genotype must be able to influence the assotment of microbes present and maintained in plant tissues and such effects must be heritable.
* Microbes must be present in seed or assocaited with seed and be able to be vertically transmitted to the next generation relaibly.

If these three processes coocure in a plant over many generations, then it is possible, if not likley, that the evolution of the plant and its microbiome could become linked and the survival of both depend upon the other.
This could result in the combination of the plant and a subset of its microbiome being effectivly a single organism in an evolutionary sense (i.e. a "holobiont"), just as mitochondria and chloroplasts are now integrated into eukaryotic cells [@sagan1967origin].
Any microbes evolving along this pattern could be thought of as hertible genetic elements, such as plasmids or plastids.
The following sections provide evidence that each of these processes do occur in seed plants, at least individually.

#### Microbial symbionts can have positive effects on plant fitness


#### The effect of plant genotype on microbiome ia a heritable trait

Many studies suggest that the genotype of plants has an effect on the composition of thier microbiomes, especially as an interaction with envrionmental factors, although the effect of envrionment alone is usually greater [@vandenkoornhuyse2015importance].
This greater effect of envrioment on the entire microbial community does preclude the possibilty that specific members of the microbiome are more influnced by genotype, as would be expected of important and coevolving symbionts, since these studies typically analyze the microbiome as whole and do not examine each taxon individually.
An effect of any size of genotype on the community as a whole indicates that the composition of the community is a heritable trait [@peiffer2013diversity] that can influence plant evolution to the degree that the identity of symbiontic microbes has an effect on plant fitness.
The classical examples of mychorrizal fungi aiding in phosphorus absorption and rhizobia aiding in nitrogen fixation indicate that specific members can have a large effect, even if most members of the microbiome of these plants is transient and facaltative.

A study on the microbiome of tomatoe seeds indicated that the two clutivars studied had diffetent microbial communites when grown in controled condions in the same soil [@bergna2018tomato].


#### Microbes can be vertically transmitted via seed

Evidence is accumulating that microbes present in the seeds of some plant species can innoculate the next generation and thus potentially sustainning the assocaiton indefinitly.
Although this is not a new observation [@boursnell1950symbiotic], its has rapidly gained attention in the last decade [@nelson2018seed], due to culture-independint technique like metabarcoding revealing unexpectedly diversre communites of symbiotic microbes in other plant organs [@lundberg2012defining].
In the 1940's, researchers desrbied observing potentially symbiotic fungi being transmitted between generation via propagules on the seed coats of the she-oak *Casuarina equisetifolia* [@bose1947hereditary] and the rock-rose *Helianthemum chamaecistus* [@boursnell1950symbiotic].
In a 2003 study, authors claim to have demonstrated vertical trasmission of benificial *Stagnospora* species via seed in the reed grass *Phragmites australis* by innoculating steril plants and measuring plant biomass [@ernst2003endophytic]. 
For all the above cases, the fungus was thought to colonize the roots and travel up the plant as it grows.
Many reserachers of grass endophytes belive the endophytes like *Epichole*  inhabiting grasses like *Lolium* to be vertically transmitted via seed and benifical to plant growth [@latch1985endophytic; @an1993relationships; @clay1990fungal; @clay1987effects; @nelson2018seed].
A recent study of bacterial vertical transmission in tomato found that at least three benifical bacteria species were preserved over two generations via seed [@bergna2018tomato].
Another recent study found that a core seed micrbiome of at least 8 fungi and bacteria was maintained over three generations of radish, although most of the microbiome was variable [@rezki2018assembly].

#### Endophytes of grasses


#### Endophytes of conifers

Much less research has been done on conifer seed endophytes.
Most of the research that has been done was based on culturing from surface-sterilized seeds, so the results will likely vastly underestimate the true diversity of micrbos that could be vertically-trasmitted.
@newcombe2018hypothetical reports 41% of seeds collected from *Pinus ponderosa* and *Larix occidentalis* contained one or more endophytes, as determined by culturing surfaced-sterilized seeds on PDA.
@ganley2006fungal found 16 isolates in 800 surface-sterilized seeds of *Pinus monticola* using culturing on PDA with antibiotics that selected for fungi.
Putativly benificial *Pseudomonas* and *Rahnella* isolates were found in the seeds of 3 out of 4 trees of *Picea abies* and confirmend by PCR [@cankar2005bacterial].

#### Metabarcoding can circumvent the limitations of past research

Most research in seed microbes has concluded that many, if not most, seeds are microbe-free.
However, such obervations would be expected even if that was not true due to the techniques used.
Any microbes that are tightly integreated with thier plant host are unlikly to be culturable, such as obligate pathogens or symbionts.
Therefore, studies that rely on culturing microbes from seeds in axenic conditions cannot be used to test for the presence of such microbes or their effect on plant fecundity.
Additionally, the number of culturable microbes present in seeds decreses in storage [@cankar2005bacterial], so studies that cluture microbes from seeds stored in seed banks or from commercial sources will likley underestime endophyte diversity and abundance.
Many studies do not report how long seeds were stored between collection and culturing and that information is even less avaialble when the seeds are bought from commerical or govermnet seeds banks.
Finally, it is common practice to sterilize the outside of the seed in order to insure what is cultured is inside the seed.
However, a vertically transmitte symbiont does not need to be inside the seed in order to colonize the germinating seedling.
In fact, some of the earliest observations of seed endophytes report endophytes on the seed coat that colonized the growing plants through the roots after germination [@boursnell1950symbiotic; @bose1947hereditary].
All of these biases limit the use of most previous research in eveaulating the potential for non-culuturable obligate symbionts in seeds that might occur on the seed coat. 
Recent adavnced in high-thoughput sequencing now allow these limitations to be overcome, so it is appropriate to revisit the topic of seed endophyte frequency with these new tools. 

#### The holobiont hypothesis


### Specific aims

1. Determine which, if any, microbial taxa show patterns of occurance and similarity in confier and grass seeds that are consistent with long-term coevolution with their host, mediated by vertical transmission via seed.
1. Characterize the bacterial, fungal, and oomycete diveristy assocaited with conifer seeds and grass seeds, including unculturable microbes and microbes on seed coat.
1. Determine the relative influence of plant species and envrionemnt on the composition of  bacterial, fungal, and oomycete commnities in individule confier and grass seeds, including unculturable microbes and microbes on seed coat.
1. Provide bacterial, fungal, and oomycete diveristy data for confier and grass seeds, tissues, and assocaited soils for two LTER site that can inform future research.


### Hypotheses

For any given microbal taxon, the following lifestyles are possible in regards to seeds:

1. Absent: The taxon does not occur in seeds.
1. Coincidental: Occurance appears random and does not correlate with either 
2. Opputunistic: The microbal taxon occurs regulary in seeds and its frequency correlates with a combination of plant species and evironmental factors. Evolution might select for certain interactions, but selection pressures on plants and the microbal taxon are mostly independent and any symbioses formed are temporary and interchangable. Such microbes are opputunistic symbionts, with effects on the host that drift between positive and negative and time and conditions change. 
3. Obligate: The microbial taxon occurs in a subset of seeds and its occurance correlates with plant species, but it evolves independently. Such microbes are obligate symbionts, but are not vertically transmitted.
4. Integrated: The microbal taxon is effectivly vertically transmitted and coevolves with the plant and selection pressure on the host and the microbes is linked. The lifestyle of these microbes might range from obligate pathogens to mutulaists.


Patterns:

1. Absent: The taxon does not occur in seeds.
1. Coincidental: Similiarty/occurance of communities of the taxon do not correlate with envrionment or host phylogeny. 
2. Opputunistic: Similiarty/occurance of communities of the taxon correlate with envrionment, but not host phylogeny.
3. Obligate: Similiarty/occurance of communities of the taxon correlate with both envrionment and host phylogeny.
4. Integrated: The microbal taxon is effectivly vertically transmitted and coevolves with the plant and selection pressure on the host and the microbes is linked. The lifestyle of these microbes might range from obligate pathogens to mutulaists.



### Broader impacts


### Methods

#### Sampling summary

Two mojor groups of plants will be sampled from two LTER sites: Confiers in the HJ Andrews forest in Oregon and grasses in the Konza prarie in Kansas.
At each site, 10 species of plant will be sampled, but only the 9 most sucessfully sampled species from each site will be used in the final analysis.
For each species, 8 plants will be sampled, each at a different part of the site.
For each plant sampled, 4 individual seeds will be collected and a mixture of 40 seeds.
The will result in a total of `r sites` x `r species` x `r trees` x `r seeds` = `r sites * species * trees * seeds` samples analyzed.


#### Species targeted

Species to be studied were selected based on abundance at their respective LTER sites, abundance of seed production, similarity of seasonal times of seed maturity, and a range of inter-species taxonomic similarity.
Abundance of species at the study sites were determined using publicly available datasets describing the LTER sites to be sampled [@mckee2001vascular; @shaw2001long; @hartnet2019plant].
Seasonal times of seed maturity were determined using species reviews in the USDA Fire Effects Information System (FEIS) database [@tesky1992tsuga1; @tesky1992tsuga2; @cope1992abies; @cope1993abies; @cope2000abies; @uchytil1991pseudotsuga; @uchytil1991picea; @griffith1992picea; @tesky1992thuja; @tollefson1992calocedrus; @tirmenstein1990taxus; @uchytil1988andropogon; @walkup1991sorgastrum; @walkup2010sporobolus; @steinberg2002schizachyrium; @chadwick2003bouteloua; @simonin2000koeleria; @snyder1992sporobolus; @simonin2000elymus].
All species typically produce mature seeds between August and October, with the majority maturing in September.
The conifer species typically produce large numbers of cones in intervals of 1-4 years, with little cone production in intervening years. 
This poses a challenge to collecting cones from all species in a single year and makes it unlikely that all species can be sampled over a short period of time.
Grass species selected are all warm-season grasses that produce seeds yearly between July and September, depending on weather.
This makes it likely that all grasses can be sampled in a single season.
Due to the complexity of sampling multiple species with potentially differing times of seed maturation, it is assumed that it might not be possible to sample all 10, so only the 9 from each group that are sampled most completly will be used for the final analysis.

Conifers: *Tsuga heterophylla*, *Tsuga mertensiana*, *Abies amabilis*, *Abies procera*, *Abies grandis*, *Pseudotsuga menziesii*, *Picea engelmannii*, *Pinus monticola*, *Thuja plicata*, *Calocedrus decurrens*, *Taxus brevifolia*

Grasses: *Andropogon gerardii*, *Sorghastrum nutans*, *Sporobolus compositus*, *Sporobolus heterolepis*, *Schizachyrium scoparium*, *Bouteloua curtipendula*, *Panicum virgatum*, *Koeleria macrantha*, *Eragrostis spectabilis*, *Elymus canadensis*


#### Site and plant selection

Sites to sampled will be selected to strike a balance between maximizing envrionmental heterogenetity amoung sites and maximizing number of target species present per site.
Differences in envrioment is needed to differentiate between diffential selection of microbes from the envrionment versus an adaptive seed microbiome. 
By contrast, having many target species in the same site allows for envrioment to be controlled, in order to best detect differences in community structure that correlate to plant type, whatever their cause.

Sampling sites at HJ Andrews will be selected ahead of time using existing pre-existing datasets containing tree species and locations in research plots.
Sampling sites at Konza will be randomly selected from a larger set of sites along one or more random transects that meet the site criteria described above.

#### Seed collection techniques

Seed collection techniques chosen are a compromise between minimizing environmental contamination and maximizing seed maturity.
Ideally seeds would be collected at the instant they are dispersed from the plant, since this is the stage at which they would have their entire potential microbiome derived from the parent, but have not been exposed to other environmental microbes.
However, waiting for a given plant to disperse seeds is not feasible in a study of this scale, so seeds will be cnough cones per direction will be sampled to ensure at least 10 large seeds can be extracted, as determined by typical numbers of seeds per cone and dissections of cones in the field.
Later, on same day as collections, seeds will be extracted from cones using sterile forceps.
A single seed will be collected from each cone representing each cardinal direcollected when nearly mature.

Cones from conifers will be collected by gloved hands when possible and knocked down using a forestry slingshot and caught when too high.
Only cones that do not contact the ground will be used, to minimize contamination.
When possible, cones from each cardinal direction will be sampled and placed in a bag for each direction sampled.
Etion on each tree and placed individually in small zip-lock bags.
Up to ten seeds from each direction will then be extracted and bulked together per tree, for a total of 40 seeds per tree.
Therefore each tree will have 4 individual seeds from each cardinal direction and a single bulked sample. 
All seeds will be frozen within 24 hours of collection.

Grass seedheads will be collected in a way analogous way to cones in conifers.
Four seedheads per grass cluster will be collected and put in individual bags.
Later, on same day as collections, seeds will be extracted using sterile forceps.
A single seed will be collected from each head and placed individually in small ziplock bags.
Up to ten seeds from each head will then be extracted and bulked together per sampled site, for a total of 40 seeds per site.
All seeds will be frozen within 24 hours of collection.


#### Envriomental data

Various indicators of environmental conditions will be measured for each plant sampled in order to best approximate envrionmental condtions that might influence the seed microbiome.
In the context of the seed, anything outside of the seed, including the surrounding plant tissue, can be considered the envrioment.
Although it is not possible the fully capture the complexity of the environment, the following aspects will be recorded to approximate envrionmental differences:

* A smaple of tissue near each cone/grasshead and vasular tissue leading to the cone/grasshead will be collected with gloves and bulked per plant. The plant tissue will not be surfaced sterilized, so that microbes that colonize the plant surface from air or precipitation will be present, as well as endophytes.
* A soil sample will be collected for each plant, composed of four 1x20cm soil cores taken in cardinal directions around the plant and bulked and well mixed.
* The elevation, slope, aspect of the slope, and convexity of the landscape will be recorded for each plant by mapping the GPS locations of plants sampled and comparing it to topographic data.
* Statistics summarizing weather in the two years precceding the time each plant was sampled will be looked up from public databases. This will only be relevant if all plants cannot be sampled in the same year. A two year period was chosen since some of the conifer species take two years to produce mature cones.  

Plant tissue and soil samples will be subjected to metabarcoding in the same way as seeds, except they will be allowed more sequencing space to characterize their more complex communities effectivly. 
Aspects relating to climate will not be considered, since this is assumed to be effectivly the same withing each LTER site during the study period and the two LTER sites are not being compared to eachother.


#### Metabarcoding

Metabarcoding of bacterial, fungal, and oomycete communities of seeds and envrionmental samples will be conducted using the Illumina MiSeq sequencer at the Oregon State University Center for Genome Research and Biocomputing (CGRB).
DNA from seed and other plant tissue samples will be extracted using the Qiagen DNeasy PowerPlant Pro Kit and DNA from soil samples will be extracted using the Qiagen DNeasy PowerSoil Kit.
Negative DNA extraction controls will be used to account for any contamination.
Each DNA extract will be used in three different PCR reactions to amplify: the V4 region of the 16S ribosomal gene of bacteria [@walters2016improved], the ITS1 region of fungi, and the Rps10 gene of oomycetes.
The products of these three reactions will then be pooled per-sample in the following proportions: 50%, 40%, 10% for bacteria, fungi, and oomycetes respectivly, so the sequencing effort roughly corresponds to expected organismal diversity.
Negative PCR controls will be used for all reactions, in addition to the negative DNA extraction controls to help determine the source and content of any contamination that might occur.
Multiple identical mock community samples composed of a mixture known bacterial, fungal, and oomycete DNA extracts will also be amplified with each reaction as a positive control and in order to asses errors rates of sequence identification and realative abundance.   
For each LTER site, `r species * trees * seeds` seed PCR products will be sequenced with `r seed_neg_dna_controls` negative DNA extraction controls, `r seed_neg_pcr_controls` negative PCR controls, and `r seed_mock_controls` mock community samples, divided into four 96-well plates containing such that each plate contains an equal number of samples and controls.
All of the seed samples and assocaited controls will be sequenced on a single MiSeq run for each LTER site.
Similarly, all plant tissue and soil samples will be sequenced on a single MiSeq run, together with 4 negative DNA extraction controls, 4 negative PCR controls, and four mock communities, for each LTER site.
Overall `r samples_sequenced` samples will be sequenced on four 300bp paired-end miseq runs.


#### Data preparation

The entire data analysis, from raw MiSeq reads to publication figures, will be conducted in R using a powerful desktop computer.
Sequence quality filtering, infferece of Extact sequence variants (ESVs), read mearging, chimera removal, and taxonomy assignment will be done using the R package `dada2`.
Standard ecological metircs of diversity such alpha and beta diversity will be calculated for each sample and grouping of samples (e.g. plant species) using the R pacakge `vegan`.
General data manipulation and plotting of data will be done use the R pacakges `dplyr`, `ggplot2`, `taxa`, and `metacoder`, among others.  

#### Hypothesis testing


#### Reproducibility

All analysis code will be written in Rmarkdown in such a way that the entire analysis, from raw reads to publication figures, can be run with a single command.

\newpage

## Biographical Sketch(es)

## Budget and Budget Justification

#### DNA extraction


```{r}
(soil_kit_dna_controls <- soil_neg_dna_controls * sites)

(plant_kit_dna_controls <- (seed_neg_dna_controls + plant_neg_dna_controls) * sites)

(soil_kit_rxns <- soil_samples +  soil_kit_dna_controls)
(plant_kit_rxns <- plant_samples + seed_samples + plant_kit_dna_controls)

soil_kit_price <- 567
soil_kit_count <- 100
(soil_kits_needed <- ceiling(soil_kit_rxns / soil_kit_count * estimate_error))

plant_kit_price <- 236
plant_kit_count <- 50
(plant_kits_needed <- ceiling(plant_kit_rxns / plant_kit_count * estimate_error))
```


* Qiagen DNeasy PowerSoil Kit (`r soil_kit_count`) x`r soil_kits_needed`: \$`r soil_kit_price * soil_kits_needed`
* Qiagen DNeasy PowerPlant Pro Kit (`r plant_kit_count`) x`r plant_kits_needed`: \$`r plant_kit_price * plant_kits_needed`
* Pipette filter tips: ~\$3000

#### PCR

```{r}
loci <- 3

(all_controls <-  (seed_neg_dna_controls +  
    seed_neg_pcr_controls +  
    seed_mock_controls + 
    soil_neg_dna_controls + 
    soil_neg_pcr_controls + 
    soil_mock_controls + 
    plant_neg_dna_controls + 
    plant_neg_pcr_controls + 
    plant_mock_controls) * sites)
(total_pcr_rxns <- (seed_samples + soil_samples + plant_samples + all_controls) * loci)

rxn_volume <- 15 # 5uL for gel, 8uL for sequencing, 2uL for error
(rxn_per_kit <- ceiling(500 * 25/15))

(pcr_kits_needed <- ceiling(total_pcr_rxns / rxn_per_kit * estimate_error))

pcr_kit_cost <- 525
```


* KAPA HiFi DNA Polymerase (500 x 25 ÂµL) x `r pcr_kits_needed`: \$`r pcr_kits_needed * pcr_kit_cost`
* Pipette filter tips: ~\$3000
* 96 well PCR plates and caps: ~\$300

#### Sequencing


* Nextera XT DNA Library Preparation Kit (96) x `r ceiling(samples_sequenced / 96)`: `r ceiling(samples_sequenced / 96) * 2943`
* Nextera XT Index Kit x 4:  \$`r 918 * 4`
* MiSeq reagents and run x4: \$`r 1917 * 4`

#### Konza collecting trip

* Site use and housing: \$33/person/day x 60: $2000
* Transportation: \$1000

#### H.J. Andrews collecting trip(s)

* Camping: \$5/night/person x30: $150
* Transportation: \$200
* Bigshot slingshot: \$150

#### Personel

* Postdoc salery and benifits: $54,000 x 4 years: $216,000

## Current and Pending Support

## Facilities, Equipment and Other Resources

## Data Management Plan

## References Cited






## Quick notes

* If vertical transmision is a stable process,  you would expect to see groups of related microbial taxa shared amoung related plants rather than the exact same microbial species.

